engine/installation/linux/docker-ce/debian.md

```diff
@@ -23,7 +24,7 @@ and distributions for different Docker editions, see
 
 ### OS requirements
 
-To install Docker, you need the 64-bit version of one of these Debian or
+To install Docker CE, you need the 64-bit version of one of these Debian or
```

Comment:
For uninstall older versions, also may be good to add `docker.io` to make sure the debian packaged version is not installed https://packages.debian.org/sid/docker.io